config/overlays/cluster/patch-add-env-var-kube-adaptor.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: skupper-controller
+spec:
+  selector:
+    matchLabels:
+      application: skupper-controller
+  template:
+    spec:
+      containers:
+        - name: controller
+          env:
+            - name: SKUPPER_KUBE_ADAPTOR_IMAGE
```

Comment:
Suggest that the image env vars be part of base as the csv bundle will also need. Then the patch for the "RELATED_IMAGES" can be carried in the config/manifests if that makes sense.